lib/sedna/core/joint_inference/joint_inference.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from copy import deepcopy
+
+from sedna.common.utils import get_host_ip
+from sedna.common.class_factory import ClassFactory, ClassType
+from sedna.service.server import InferenceServer
+from sedna.service.client import ModelClient, LCReporter
+from sedna.common.constant import K8sResourceKind
+from sedna.core.base import JobBase
+
+__all__ = ("JointInference", "TSBigModelService")
+
+
+class TSBigModelService(JobBase):
+    """
+    Large model services implemented
+    Provides RESTful interfaces for large-model inference.
+    """
+
+    def __init__(self, estimator=None, config=None):
+        super(TSBigModelService, self).__init__(
+            estimator=estimator, config=config)
+        self.local_ip = self.get_parameters("BIG_MODEL_BIND_IP", get_host_ip())
+        self.port = int(self.get_parameters("BIG_MODEL_BIND_PORT", "5000"))
+
+    def start(self):
+        if callable(self.estimator):
+            self.estimator = self.estimator()
+        if not os.path.exists(self.model_path):
+            raise FileExistsError(f"{self.model_path} miss")
+        else:
+            self.estimator.load(self.model_path)
+        app_server = InferenceServer(model=self, servername=self.job_name,
+                                     host=self.local_ip, http_port=self.port)
+        app_server.start()
+
+    def train(self, train_data,
+              valid_data=None,
+              post_process=None,
+              **kwargs):
+        self.log.warn("Joint inference Experiment should offer model file")
```

Comment:
1. Why does joint-inference have train?
2. Experiment means ?
3. Why does warn at the very firstly of train